docs/spec.md

```diff
@@ -1364,16 +1364,16 @@ in the `operand` tensor and produces a `result` tensor. More formally,
 
 #### Inputs
 
-| Label | Name                   | Type                                         | Constraints   |
-|-------|------------------------|----------------------------------------------|---------------|
-| (I1)  | `operand`              | tensor or per-tensor quantized tensor        | (C1-C2), (C5) |
-| (I2)  | `broadcast_dimensions` | 1-dimensional tensor constant of type `si64` | (C2-C5)       |
+| Label | Name                   | Type                                         | Constraints      |
+|-------|------------------------|----------------------------------------------|------------------|
+| (I1)  | `operand`              | tensor or quantized tensor                   | (C1-C2), (C5-C6) |
+| (I2)  | `broadcast_dimensions` | 1-dimensional tensor constant of type `si64` | (C2-C6)          |
 
 #### Outputs
 
-| Name     | Type                                  | Constraints      |
-|----------|---------------------------------------|------------------|
-| `result` | tensor or per-tensor quantized tensor | (C1), (C3), (C5) |
+| Name     | Type                       | Constraints         |
+|----------|----------------------------|---------------------|
+| `result` | tensor or quantized tensor | (C1), (C3), (C5-C6) |
 
 #### Constraints
```

Comment:
Actually, following-up on Doyeon's comment at https://github.com/openxla/stablehlo/pull/1747#discussion_r1302338221, I think (C1) should be broken down into two separate conditions, one for quantized type and non-quantized type.

The reason why we need a separate issue for the quantized type is that the `element_type` is considered different when at least one of the parameter is different. If two quantized types have different quantization axes then they are considered different, e.g. `!quant.uniform<i8:f32:0, {0.1:-30,0.5:-20}> != !quant.uniform<i8:f32:1, {0.1:-30,0.5:-20}>`. If an op modifies the quantization params then C1 is violated.

I think this should be applied to all ops that **may** modify the quantization parameters.

Reply:
Thanks! LGTM.